tests/playwright/sanity/modules/nested-tabs/nested-tabs-atomic-repeaters.test.ts

```diff
@@ -94,14 +94,13 @@ test.describe( 'Nested Tabs experiment is active @nested-atomic-repeaters', () =
 		await editor.selectElement( nestedTabsID );
 
 		await test.step( 'Add multiple items using repeater', async () => {
-			// Add 10 items to the repeater
-			for ( let i = 0; i < 4; i++ ) {
+			for ( let i = 0; i < 2; i++ ) {
```

Comment:
Adding many repeater items on the page caused a problem. Possibly because the controls were displayed outside of the page.